sdk/search/Azure.Search.Documents/tests/Batching/BatchingTests.cs

```diff
@@ -180,11 +181,19 @@ private void RaiseNotification()
 
             public override Response<IndexDocumentsResult> IndexDocuments<T>(IndexDocumentsBatch<T> batch, IndexDocumentsOptions options = null, CancellationToken cancellationToken = default)
             {
+                using DiagnosticScope scope = ClientDiagnostics.CreateScope($"{nameof(BatchingSearchClient)}.{nameof(IndexDocuments)}");
```

Comment:
These need to go in the client - not the test. The records probably aren't matching up.

Reply:
Actually, since @tg-msft is deriving a client I suppose this makes sense here. Sorry I missed that. The limited view GitHub shows made me think this was tests for batching. Only later did I remember reviewing this when @tg-msft first requested a merge.

Still, I'm not sure why you're seeing all the failures. I don't see anything in this PR to account for that. I recommend running tests locally (well, always) and stepping through a failure. Nothing comes to mind, unfortunately.

Reply:
That's the thing though, the tests are passing locally. :(

Reply:
Did you find a bunch of *.json files you need to `git add`? I see a few here, but perhaps not enough. And do you have `AZURE_TEST_MODE` set to `Record` or `Live`, perhaps that would affect your VS instance? Generally we try to stay away from that variable and re-record per-class (less messy).

Reply:
I also ran the tests locally and they passed for me.. The bizarre thing is that the tests being run from CI on this PR don't appear to be the same test files as are currently committed - the line numbers in the error messages are not lining up.